src/parser/rogue/outlaw/modules/spells/BetweenTheEyes.tsx

```diff
@@ -0,0 +1,117 @@
+
+import { t } from '@lingui/macro';
+
+import { NumberThreshold, ThresholdStyle, When } from 'parser/core/ParseResults';
+
+import Events, { EventType, UpdateSpellUsableEvent } from 'parser/core/Events';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+
+import AlwaysBeCasting from '../features/AlwaysBeCasting';
+
+class BetweenTheEyes extends Analyzer {
+
+  timestampFromCD: number = 0;
+  totalTimeOffCD: number = 0;
+  isFirstCast: boolean = true;
+
+  static dependencies = {
+    spellUsable: SpellUsable,
+    alwaysBeCasting: AlwaysBeCasting
+  };
+  protected spellUsable!: SpellUsable;
+  protected alwaysBeCasting!: AlwaysBeCasting;
+
+  constructor(options: Options) {
+    super(options);
+    this.addEventListener(Events.UpdateSpellUsable.by(SELECTED_PLAYER).spell(SPELLS.BETWEEN_THE_EYES), this.onBetweenTheEyesUsable);
+  }
+
+  onBetweenTheEyesUsable(event: UpdateSpellUsableEvent) {
+    switch (event.trigger) {
+
+      case EventType.BeginCooldown: {
+
+        if (!this.isFirstCast) { 
+          this.totalTimeOffCD += event.timestamp - this.timestampFromCD;
+        } else {
+          this.isFirstCast = false;
+        }
+        break;
+      }
+        
+      case EventType.EndCooldown: {
+        this.timestampFromCD = event.timestamp;
+        break;
+      }
+    }
+  }
+
+  // Thresholds get retrieved at the end of analyzing
+  get thresholds(): NumberThreshold {
+    return {
+      actual: this.percentTimeOnCD,
+      isLessThan: {
+        minor: 0.9,
+        average: 0.80,
+        major: 0.75,
+      },
+      style: ThresholdStyle.PERCENTAGE,
+    };
+  }
+
+  get secondsOffCD(): number {
+    return (this.totalTimeOffCD / 1000);
+  }
+
+  get percentTimeOffCD(): number {
+    return this.secondsOffCD / (this.owner.fightDuration / 1000);
+  }
+
+  get percentTimeOnCD(): number {
+    return 1 - this.percentTimeOffCD;
+  }
+
+  get inefficientCastSuggestion(): JSX.Element {
+    return <>You should try to cast <SpellLink id={SPELLS.BETWEEN_THE_EYES.id} /> more often.
+      <SpellLink id={SPELLS.BETWEEN_THE_EYES.id} /> should almost always be used as a finisher when it is available</>;
+  }
+
+  statistic() {
+    return (
+      <Statistic
+        size="flexible"
+        category={STATISTIC_CATEGORY.GENERAL}
+        tooltip={<>This is the time of how much of the fight <SpellLink id={SPELLS.BETWEEN_THE_EYES.id} /> was on cooldown. <SpellLink id={SPELLS.BETWEEN_THE_EYES.id} /> generally has to be used as soon as it comes off cooldown, cast should therefore only be delayed for a minimum amount of time in order to maximise debuff uptime </>}
+      >
+        <BoringSpellValueText spell={SPELLS.BETWEEN_THE_EYES}>
+            <>
```

Comment:
I reckon this fragment is unnecessary (but perhaps `BoringSpellValueText`'s typing is too strict).